taxi/views.py

```diff
@@ -57,7 +79,22 @@ class ManufacturerDeleteView(LoginRequiredMixin, generic.DeleteView):
 class CarListView(LoginRequiredMixin, generic.ListView):
     model = Car
     paginate_by = 5
-    queryset = Car.objects.select_related("manufacturer")
+
+    def get_context_data(self, object_list=None, **kwargs):
+        context = super().get_context_data(**kwargs)
+        model = self.request.GET.get("model", "")
+        context["car_search_form"] = CarsSearchForm(
```

Comment:
you could rename all the search forms with same name in context and put this form into includes in the templates so you don't have to repeat